src/rank_llm/demo/rerank_dataset_with_prebuilt_index.py

```diff
@@ -0,0 +1,23 @@
+import sys
+import os
+
+SCRIPT_DIR = os.path.dirname(os.path.abspath(__file__))
+parent = os.path.dirname(SCRIPT_DIR)
+parent = os.path.dirname(parent)
+sys.path.append(parent)
```

Comment:
Is there any work around to having this in each file?

Reply:
Most likely, there is. I looked around for a bit, could find any examples. let's leave it as is for now since it is not a blocker for sigir release. I filed an issue for tracking